xcp_d/cli/parser.py

```diff
@@ -323,7 +337,14 @@ def _build_parser():
         default='auto',
         choices=['y', 'n'],
         action=parser_utils.YesNoAction,
-        help='After denoising, concatenate each derivative from each task across runs.',
+        help=(
```

Comment:
this is the main request, right?

Reply:
I started trying to fill in these help strings before I started working on the request, but they're linked.